index.js

```diff
@@ -168,10 +174,6 @@ const run = async () => {
 
 const check = () => {
   return new Promise((resolve) => {
-    if (!url.includes("https://github.com/")) {
```

Comment:
you removed this, add some message for invalid url na @Tanmay000009

Reply:
Moved the updated logic at line 139

Reply:
gg